SourceCode/Power/ftt_p_main.py

```diff
@@ -810,6 +808,16 @@ def solve(data, time_lag, iter_lag, titles, histend, year, domain):
             data['MWIY'][:, :, 0] = (data_dt['MWIY'][:, :, 0]
                                      + (data['MEWI'][:, :, 0] * dt * data['BCET'][:, :, c2ti['3 Investment ($/kW)']] / 1.33))
             
+            sector_coupling_assumps = get_sector_coupling_dict(data, titles)
+            
+            battery_lifetime = 12 #assuming 12 years
+            
+            capacity_batteries_current_year = data["MSSC"] * sector_coupling_assumps["GW to GWh"]
+            capacity_batteries_last_year = time_lag["MSSC"] * sector_coupling_assumps["GW to GWh"]
+            
+            single_year_capacity = capacity_batteries_current_year - capacity_batteries_last_year
+            
+            yearly_replacements_existing_capacity = single_year_capacity + data["MSSC"]/battery_lifetime
```

Comment:
Again: make sure you choose a specific variable name. This is overall additions, not only replacements. 

More importantly: you need to use "capacity_batteries_current_year" rather than data["MSSC"], as data["MSSC"] is in the wrong units.